internal/run/convert.go

```diff
@@ -0,0 +1,104 @@
+package run
+
+import (
+	"errors"
+	"fmt"
+	"log"
+
+	convertcmd "github.com/helm/helm-2to3/cmd"
+	"github.com/helm/helm-2to3/pkg/common"
+	"github.com/pelotech/drone-helm3/internal/env"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/storage/driver"
+)
+
+func v3ReleaseFound(release string, cfg *action.Configuration) bool {
+
+	// check if release exists
+	_, err := cfg.Releases.Deployed(release)
+	switch {
+	// case errors.Is(err, driver.ErrReleaseNotFound), errors.Is(err, driver.ErrNoDeployedReleases):
+	case errors.Is(err, driver.ErrReleaseNotFound):
+		log.Printf("No v3 Release of %s found", release)
+	case err == nil:
+		log.Printf("A v3 Release of %s was found", release)
+		return true
+	}
+
+	return false
```

Comment:
**nit:** Seems like you could simplify this to the following to make it a bit more readable.

```
if _, err := cfg.Releases.Deployed(release); err == nil {
	log.Printf("A v3 Release of %s was found", release)
	return true
}

log.Printf("No v3 Release of %s found", release)
return false
```

Reply:
done